hack/.spelling_failures

```diff
@@ -1,5 +1,6 @@
 events/elections/2017/
 vendor/
+sig-docs/meeting-notes-archive/
```

Comment:
Is there a reason for this? This spell check is to prevent folks coming in later and PRing in spelling fixes

Reply:
The first PR I made had found 3 spelling errors that I had to resolve manually. I figured this would make it easier for meeting minutes 2020+ to be added without fuss.

I'm ok with removing it if you feel strongly against it.

Reply:
Yes, the spell checker was put in place to keep PR volumes against this repo manageable. Fixing three spelling errors at the time of PR is preferable.